### PR TITLE
API 15094 Remove Unsupported Environments

### PR DESCRIPTION
**Pull request by agarcia-va**

### CODE REVIEW

**Comment by taylorrassmannlibertyits** [comment]
Was this meant to be a draft PR?

**Comment by agarcia-va** [comment]
> Was this meant to be a draft PR?

Yes. Originally I had thought this story would be split into two PRs, and opened this one under that expectation. After being corrected I switched to a draft PR until I could add the small update that has just been added. It is a full PR now!

**Review by taylorrassmannlibertyits — DISMISSED** [review]

File: provider-directory-tests/src/main/docker/entrypoint.sh [hunk 1]
```diff
@@ -5,11 +5,11 @@ set -euo pipefail
...
 
-test -n "${K8S_ENVIRONMENT}"
-test -n "${K8S_LOAD_BALANCER}"
+test -n "${DEPLOYMENT_ENVIRONMENT}"
+test -n "${DEPLOYMENT_TEST_HOST}"
 
-if [ -z "${SENTINEL_ENV:-}" ]; then SENTINEL_ENV="$K8S_ENVIRONMENT"; fi
-if [ -z "${SENTINEL_URL:-}" ]; then SENTINEL_URL="https://${K8S_LOAD_BALANCER}"; fi
+if [ -z "${SENTINEL_ENV:-}" ]; then SENTINEL_ENV="$DEPLOYMENT_ENVIRONMENT"; fi
+if [ -z "${SENTINEL_URL:-}" ]; then SENTINEL_URL="${DEPLOYMENT_TEST_PROTOCOL}://${DEPLOYMENT_TEST_HOST}:${DEPLOYMENT_TEST_PORT}/provider-directory"; fi
```
[the remark this answers is not in the document]
Comment (agarcia-va) [reply provider-directory-tests/src/main/docker/entrypoint.sh:12]: `https://${K8S_LOAD_BALANCER}` should point to an `https` url for the load balancer. `${DEPLOYMENT_TEST_PROTOCOL}://${DEPLOYMENT_TEST_HOST}:${DEPLOYMENT_TEST_PORT}/provider-directory` should point to the same, plus the defined port and `/provider-directory`. Unless the D2 definitions in the deployer are incorrect, then this should match.

[the remark this answers is not in the document]
Comment (jblefkove-va) [reply provider-directory-tests/src/main/docker/entrypoint.sh:12]: What are the values after you substitute the environment variables?

**Review by jblefkove-va — APPROVED** [review]

**Pull request merged**